src/types/grid.d.ts

```diff
@@ -38,3 +38,9 @@ type GridRows = {
 };
 
 type Grid = GridColumns | GridComponent | GridGutter | GridRows;
+
+type DataGridColumnDef = import('ag-grid-community').ColDef;
+type TRowData = any;
```

Comment:
Just to stay consistent with the rest of the typings file, can we add newlines between these types?